src/styles/view-switcher.scss

```diff
@@ -0,0 +1,61 @@
+.view-switcher {
+  display: flex;
+  justify-content: center;
+
+  // These are the exact margins required to vertically align the top of the
+  // view switcher with the top of the Hide Highlights button to its left,
+  // and the top of the first annotation card below the view switcher with the
+  // top of the New Page Note button to its left.
+  margin-top: 1px;
+  margin-bottom: 6px;
+}
+
+.view-switcher__tab {
+  @include smallshadow
+
+  height: 30px; // Same height as used for Hide Highlights and New Page Note
+                // buttons to left of view switcher.
+  padding-left: 12px;
+  padding-right: 12px;
+
+  background: $white;
+  transition: background-color .1s linear;
+
+  border: 1px solid $gray-lighter;
+
+  touch-action: manipulation;
+  cursor: pointer;
+  user-select: none;
+
+  -webkit-appearance: button;
+}
+
+.view-switcher__tab {
+  border-right-width: 0;
+}
+.view-switcher__tab:last-child {
+  border-right-width: 1px;
+}
+
+.view-switcher__tab:first-child {
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+}
+.view-switcher__tab:last-child {
+  border-top-right-radius: 4px;
+  border-bottom-right-radius: 4px;
+}
+
+.view-switcher__tab:focus {
+  outline:0 !important;
```

Comment:
It doesn't seem to do anything, removed. Also added a `-moz-focus-inner` rule that stops Firefox from drawing a botted inner border inside the button when you click on it.